packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -11,16 +11,28 @@ import getScrollParent from './getScrollParent'
 // https://github.com/rollup/rollup/issues/1267#issuecomment-446681320
 const createPopper = (
   reference: Element | _PopperJS.ReferenceObject,
-  popper: Element,
+  popper: HTMLElement,
   options?: PopperJS.PopperOptions,
 ): PopperJS => {
   const instance = new ((_PopperJS as any).default || _PopperJS)(reference, popper, {
     ...options,
     eventsEnabled: false,
   })
+
+  const originalUpdate = instance.update
+  instance.update = () => {
```

Comment:
Why the PR title says 'first render'? This fixes resizing after the first render.

Reply:
Updated 👍